hedera-node/hedera-network-service-impl/src/main/java/com/hedera/node/app/service/network/impl/serdes/MonoRunningHashesAdapterCodec.java

```diff
@@ -52,12 +54,18 @@ public int measure(@NonNull DataInput input) {
     }
 
     @Override
-    public int typicalSize() {
+    public int measureRecord(final @NonNull RecordsRunningHashLeaf recordsRunningHashLeaf) {
         throw new UnsupportedOperationException();
     }
 
     @Override
     public boolean fastEquals(@NonNull RecordsRunningHashLeaf item, @NonNull DataInput input) {
         throw new UnsupportedOperationException();
     }
+
+    @NonNull
+    @Override
+    public RecordsRunningHashLeaf parseStrict(@NonNull DataInput dataInput) throws IOException {
+        return null;
```

Comment:
Should this be calling the other parse method instead of returning null?

Reply:
done thank you